lib/OpenQA/Schema/Schema.pm

```diff
@@ -24,12 +25,22 @@ use FindBin qw($Bin);
 # after bumping the version please look at the instructions in the docs/Contributing.asciidoc file
 # on what scripts should be run and how
 our $VERSION = 24;
+our $configname="$Bin/../lib/database.ini";
 
 __PACKAGE__->load_namespaces;
 
-my @paths = ( "$Bin/../lib/database", "$Bin/../../lib/database" );
-unshift(@paths, dirname($ENV{OPENQA_CONFIG}).'/database') if ($ENV{OPENQA_CONFIG});
-__PACKAGE__->config_paths(\@paths);
+
+sub connect_db {
+    my $mode = shift || $ENV{OPENQA_DATABASE} || 'production';
+    CORE::state $schema;
+    unless ($schema) {
+        my %ini;
+        #        print("ENV{OPENQA_DATABASE_CONFIG}=$ENV{OPENQA_DATABASE_CONFIG}\n");
+        tie %ini, 'Config::IniFiles', ( -file => $ENV{OPENQA_DATABASE_CONFIG} || $configname );
+        $schema=__PACKAGE__->connect($ini{$mode});
```

Comment:
Not sure what you mean by they would already be there, I think simply linking the ini files from etc/openqa (where lib/database.ini and lib/openq.ini would be moved too) would make tests work as they do now, which means to also test the project's ini files validity. If later some specific test needs special ini file(s) just for itself, it can have them in some another t/data like directory (again linking the ones that need not to be specific).